test/testvideocapture.c

```diff
@@ -0,0 +1,750 @@
+/*
+  Copyright (C) 1997-2022 Sam Lantinga <slouken@libsdl.org>
```

Comment:
It's 2023 these days. :)

Reply:
All the files should have their copyright date updated before merging.

Reply:
done + re-added WinRT projects